builtin/credential/okta/backend.go

```diff
@@ -106,8 +109,11 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 	}
 
 	var result authResult
-	rsp, err := client.Do(authReq, &result)
+	rsp, err := shim.Do(authReq, &result)
 	if err != nil {
+		if oe, ok := err.(*okta.Error); ok {
+			return nil, logical.ErrorResponse(fmt.Sprintf("Okta auth failed: %v (code=%v)", err, oe.ErrorCode)), nil, nil
```

Comment:
nit: `fmt.Sprintf` is redundant as `ErrorResponse` assumes a format string + parameters